### PR TITLE
new rpk connect

### PR DESCRIPTION
**Pull request by Deflaimun**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/<add-your-issue-number-here>
Review deadline:

This adds the new standalone rpk connect capabilitites to docs. 
Introduced by https://github.com/redpanda-data/redpanda/pull/23117 and https://github.com/redpanda-data/redpanda/pull/23305/

## Page previews

https://deploy-preview-774--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-connect/rpk-connect/
https://deploy-preview-774--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-connect/rpk-connect-uninstall/
https://deploy-preview-774--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-connect/rpk-connect-upgrade/
https://deploy-preview-774--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-connect/rpk-connect-install/

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Review by r-vasquez — APPROVED** [review]
Thanks!

**Comment by Deflaimun** [comment]
If we need to apply any changes, cherry-pick them to https://github.com/redpanda-data/docs/pull/776

File: modules/reference/pages/rpk/rpk-connect/rpk-connect-install.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+= rpk connect install
+
+Install Redpanda Connect.
+
+This command install the latest version by default. Alternatively, you may specify a Redpanda Connect version using the `--connect-version` flag.
```
Comment (asimms41) [inline modules/reference/pages/rpk/rpk-connect/rpk-connect-install.adoc]:
```suggestion
By default, this command installs the latest version of Redpanda Connect. Alternatively, you can use the command with the `--connect-version` flag to install a specific version.
```

File: modules/reference/pages/rpk/rpk-connect/rpk-connect-install.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+You may force the installation of Redpanda Connect using the `--force` flag.
```
Comment (asimms41) [inline modules/reference/pages/rpk/rpk-connect/rpk-connect-install.adoc]:
```suggestion
To force the installation of Redpanda Connect, use the `--force` flag.
```

File: modules/reference/pages/rpk/rpk-connect/rpk-connect-install.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+rpk connect install [flags]
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--connect-version |string |Redpanda Connect version to install. For example: "4.32.0" (default "latest").
```
Comment (asimms41) [inline modules/reference/pages/rpk/rpk-connect/rpk-connect-install.adoc]:
```suggestion
|--connect-version |string |The Redpanda Connect version to install. For example: "4.32.0" (default "latest").
```

File: modules/reference/pages/rpk/rpk-connect/rpk-connect.adoc [hunk 4]
```diff
@@ -1,60 +1,28 @@
 = rpk connect
 :description: These commands let you create and manage data pipelines using Redpanda Connect.
 
-Redpanda Connect provides an ecosystem of over 220 open source connectors. 
-
-See xref:redpanda-connect:ROOT:about.adoc[Redpanda Connect].
+A stream processor for mundane tasks. See xref:redpanda-connect:ROOT:about.adoc[Redpanda Connect].
```
Comment (asimms41) [inline modules/reference/pages/rpk/rpk-connect/rpk-connect.adoc]:
```suggestion
Create and manage data pipelines using Redpanda Connect. For full details, see the xref:redpanda-connect:ROOT:about.adoc[Redpanda Connect documentation].
```

**Review by asimms41 — COMMENTED** [review]
Just made a few suggestions.

**Comment by Deflaimun** [comment]
Applied suggestions from @asimms41 and cherry picked them to #776

**Comment by Feediver1** [comment]
@asimms41 @Deflaimun Is this ready to merge? Any remaining blockers?

**Review by Feediver1 — APPROVED** [review]
lgtm

**Pull request merged**